cmd/ebrelayer/contract/Peggy.sol

```diff
@@ -570,12 +576,16 @@ contract Peggy is Processor {
         whileActive()
         returns(bytes32 _id)
     {
-         //Actions based on token address type
+        string memory symbol;
+        
+        //Actions based on token address type
         if (msg.value != 0) {
           require(_token == address(0));
           require(msg.value == _amount);
+          symbol = "ETH";
         } else {
           require(ERC20(_token).transferFrom(msg.sender, address(this), _amount));
```

Comment:
should we have an approval method to give approval to the Peggy contract?

Reply:
This would just be routing the approval transaction through the Peggy contract to the ERC20 token contract instead of sending it directly to the token contract, which seems somewhat redundant. Perhaps another option is adding support for ERC20 token approvals through truffle cli - users could complete both steps through 2 simple commands, such as `yarn peggy:approve [TOKEN_ADDRESS] [AMOUNT]` and `yarn peggy:lock [RECIPIENT] [TOKEN_ADDRESS] [AMOUNT]`. This is dependent on contract tooling integration: https://github.com/swishlabsco/peggy/pull/2

Reply:
The latter sounds good. Can we add that to the tooling + script + README?

Reply:
https://github.com/swishlabsco/peggy/pull/1/commits/76f444eae41cd4d8544d9e136cc2f6b2b80ec6f4